src/vendors/oasisscan/models/index.ts

```diff
@@ -1,14 +1,17 @@
-export * from './AccountsRow'
-export * from './DebondingDelegationRow'
-export * from './DelegationRow'
-export * from './InlineResponse200'
-export * from './InlineResponse2001'
-export * from './InlineResponse2001Data'
-export * from './InlineResponse2002'
-export * from './InlineResponse2002Data'
-export * from './InlineResponse2003'
-export * from './InlineResponse2003Data'
-export * from './InlineResponse2004'
-export * from './InlineResponse2004Data'
-export * from './OperationsRow'
-export * from './ValidatorRow'
+export * from './AccountsRow';
```

Comment:
Separate the automated changes or automated+swagger changes into its own commit

(and run lint)

Reply:
This was resolved in a separate commit.

Reply:
I will clean up commits after review process.

Reply:
done